core/arch/arm/plat-aspeed/platform_ast2600.c

```diff
@@ -59,6 +59,10 @@ register_phys_mem(MEM_AREA_IO_SEC,
 		  AHBC_BASE,
 		  SMALL_PAGE_SIZE);
 
+register_phys_mem(MEM_AREA_IO_NSEC,
+		  SCU_BASE,
+		  SMALL_PAGE_SIZE);
+
```

Comment:
All those `register_phys_mem()` fit in less than 80 characters. This is a good opportunity to rewrite the whole block as:
```C
register_phys_mem(MEM_AREA_IO_NSEC, CONSOLE_UART_BASE, SMALL_PAGE_SIZE);
register_phys_mem(MEM_AREA_IO_SEC, GIC_BASE + GICD_OFFSET, SMALL_PAGE_SIZE);
register_phys_mem(MEM_AREA_IO_SEC, GIC_BASE + GICC_OFFSET, SMALL_PAGE_SIZE);
register_phys_mem(MEM_AREA_IO_SEC, AHBC_BASE, SMALL_PAGE_SIZE);
register_phys_mem(MEM_AREA_IO_NSEC, SCU_BASE, SMALL_PAGE_SIZE);
```